lib/puppet/functions/nomad/nomad_sorted_json.rb

```diff
@@ -175,7 +177,7 @@ def simple_generate(obj)
 
 
   def default_impl(*args)
```

Comment:
I think this function is actually unused altogether and can be dropped. Only `nomad::sorted_json` is used.

Reply:
this was the result of the discussion in #9 but that function was never put in service as it where, it just seems to linger for now. i am planning of reviewing it after the initial release

Reply:
I'd suggest to drop it before a 1.0.0 since removing things is a breaking change.

Reply:
Small nit: given the changes a 0.1.0 makes sense, but that's a small thing.

Reply:
0.1.0 is fine for me too, was thinking the same thing